### DISCUSSION TITLE
Create Page and use them with  privacy settings

### DISCUSSION THREAD

**Discussion by punksinatra** [GetPublii/Publii#1619] [Help] [answered]
For the longest time when pages was not an option in Publii, I was creating regular posts and marked them as hidden so they won't show up as a blog post and use them as "static pages" something like "About" "Privacy Policy Page" etc. 

Now that pages is available, I was trying to move my "privacy policy post" as a page. I deleted the "privacy policy post" and created a new page for my privacy policy post. 

Now I am trying to use the new page for "Privacy Settings". it does see the page but when I preview or create the site and check "More detail..." it does not open the new page and shows index.html#notfound when I hover over the link

![image](https://github.com/user-attachments/assets/90a3dd1b-d6eb-47fd-b3e4-2a07d0b668ff)

**Comment by bobmitro** [comment]
Provide more deatils, what theme, what Publii version (build) etc...

**Reply by punksinatra** [reply]
Hi,

Publii: Version: 0.46.0 (build 16889)
Theme: Mercury v3.0.0.0

**Reply by bobmitro** [reply, marked as answer]
Ah, I see now that it comes from the Privacy app settings. Unfortunately, it only supports posts for now.

**Reply by punksinatra** [reply]
Oh that's why, I thought I was doing something wrong. I will just wait for an update then so it has the option to use a page. 

Thank you @bobmitro
